package io.github.ol1veirx.exception;

public class PedidoNaoEncontradoException extends RuntimeException {

    public PedidoNaoEncontradoException() {
        super("Pedido n√£o encontrado.");
    }
}
